clang/test/CXX/drs/dr25xx.cpp

```diff
@@ -83,7 +83,7 @@ using ::dr2521::operator""_div;
 
 
 #if __cplusplus >= 202302L
-namespace dr2553 { // dr2553: 18 review
+namespace dr2553 { // dr2553: review
```

Comment:
All of the deducing this issue have agreement in core and i think we should signal we implement them.